### PR TITLE
[jetbrains] Update to version 213.5744

### PR DESCRIPTION
**Pull request by corneliusludmann**
## Description
This PR updates the JetBrains backends to the latest version.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/6685

## How to test
- Go to preferences and change to IntelliJ resp. GoLang IDE and start a workspace.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **corneliusludmann** after the PR has been reviewed.

Associated issue: *#6685*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by loujaybee — APPROVED** [review]
Tested working for GoLand and IntelliJ

Thanks for looking into / applying the fix here @corneliusludmann 🚀

I'd say let's get this shipped to prod so the beta testers can continue 🙏

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: c773f892369b27fb04581f9383b0ba568e0ec1e7</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6703#pullrequestreview-805903661" title="Approved">loujaybee</a>*

Associated issue: *#6685*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [loujaybee]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
